controllers/login.js

```diff
@@ -0,0 +1,26 @@
+const logIn = require("../public/javascripts/data/login");
+const encryption = require("../public/javascripts/encryption");
+
+
+// GET request
+exports.log_in = function(req, res, next) {
+
+  res.render('login');
+};
+
+// POST request
+exports.checkLogin = function(req, res, next) {
+
+  let username = req.body.username;
+  let password = req.body.password;
+
+  if(logIn.tryLogin(username, password)) {
+    // TODO:
```

Comment:
I'd invert the if check here. If there's a failure condition (incorrect login credentials in this case) you want to fail out of the method as fast as possible. Line 25 should be the redirect of the success condition, line 21 should re-render the login page with an error message

Reply:
I see. Makes sense. Thank you.